doc/cephadm/services/monitoring.rst

```diff
@@ -128,11 +128,11 @@ Some of the advantages are:
 #. **Flexible retention policies**: with per-daemon logs, log rotation is usually set to a short interval (1-2 weeks) to save disk usage.
 #. **Increased security & backup**: logs can contain sensitive information and expose usage patterns. Additionally, centralized logging allows for HA, etc.
 
-Centralized Logging in Ceph is implemented using two new services - ``loki`` & ``promtail``.
+Centralized Logging in Ceph is implemented using two new services - ``loki`` & ``alloy``.
 
 Loki: It is basically a log aggregation system and is used to query logs. It can be configured as a datasource in Grafana. 
 
-Promtail: It acts as an agent that gathers logs from the system and makes them available to Loki.
+Alloy: It acts as an agent that gathers logs from the system and makes them available to Loki.
```

Comment:
```
* Loki is a log aggregation system and is used to query logs. It can be configured as a _datasource_ in Grafana. 
* Alloy acts as an agent that gathers logs from each node and forwards them to Loki.

```